### DISCUSSION TITLE
Can the commissioner add a spreadsheet draftboard?

### DISCUSSION THREAD

**Discussion by davidkinetisense** [110yards/fantasy#176] [Q&A] [answered]
Can the commissioner add a spreadsheet draft board?

**Comment by mdryden** [comment, marked as answer]
What do you mean by that?

**Reply by davidkinetisense** [reply]
We did our draft using a Spreadsheet.  One of my players is away and I as the commissioner needs to add his players.  The commissioner should be able to add or remove players.

**Reply by davidkinetisense** [reply]
I need to add a teams players and I do not have the login.  Can you help if I send you the players?  I need it for tomorrow morning if possible.  David

**Reply by mdryden** [reply]
If you haven't drafted yet, go into your league options and set the draft type to commissioner, then draft for everyone...that's really the only option.  There's no ability right now for the commissioner to draft for only some of the teams.

I guess the only other option is that you log in as him....he could forward you the login link (log in via email), and you could use that one time to log in.  If you opened a private window you could draft his team and yours.

**Comment by davidkinetisense** [comment]
Thanks, I was able to contact him. He is in Jamaica on honeymoon, I did not
know about commissioner draft.  That is good to know.  David
